src/app/users/users.component.html

```diff
@@ -1,5 +1,6 @@
 <mat-toolbar>
   <button mat-icon-button (click)="back()"><mat-icon>arrow_back</mat-icon></button>
+  <span i18n>Users</span>

```

Comment:
this one is from another issue so when you create new branch please make it from master branch not as branch of branch